tests/src/Kernel/Entity/OgMembershipTest.php

```diff
@@ -160,6 +167,27 @@ public function testSetNoGroupException() {
   }
 
   /**
+   * Tests that a logic exception is not thrown for groups with no owner.
+   */
+  public function testNoOwnerException() {
```

Comment:
I understand that this is fixing a bug about an exception that was unnecessarily thrown, but instead of putting emphasis on the fact that no exception should be thrown, I would just mention that it should be possible to create groups with no owner. This is a valid use case, groups without owners could be created through the API, or by importing data, or REST calls or whatever.

Maybe something like this?

```
/**
 * Test that it is possible to create groups without an owner.
 */
public function testNoOwner() {
}
```